uint8 id

geometry_msgs/Point position
geometry_msgs/Vector3 relative_velocity

int8 dynamic_property
int8 class_type
